mobile/src/common/radio-buttons-group.component.tsx

```diff
@@ -0,0 +1,154 @@
+import React from 'react';
+import { TextStyle, TouchableOpacity, View, ViewStyle } from 'react-native';
+import { StyledText } from '../override/styled-text';
+import { radioButtonsGroupStyles } from './radio-buttons-group.styles';
+
+export interface RadioButton {
+    selectionIndex: number;
+    label: string;
+
+    labelStyle?: TextStyle;
+    color?: string;
+    disabled?: boolean;
+    layout?: string;
+    selected?: boolean;
+    size?: number;
+}
+
+interface RadioButtonProps extends RadioButton {
+    onPress: (label: string) => void;
+}
+
+class RadioButtonComponent extends React.Component<RadioButtonProps> {
+
+    public render() {
+        const opacity = this.props.disabled ? 0.2 : 1;
+
+        let layout: ViewStyle = { flexDirection: 'row' };
+        let margin: ViewStyle = { marginLeft: 10 };
+        if (this.props.layout === 'column') {
+            layout = { alignItems: 'center' };
+            margin = { marginTop: 10 };
+        }
+        let textStyle: TextStyle = {
```

Comment:
const textStyle = StyleSheet.flatten([
...
]);
all styles should use StyleSheet.flatten/create, it optimizes the passing ones through the bridge

Reply:
Flattening ruins optimization.
https://github.com/robinpowered/glamorous-native/issues/5
"By using StyleSheet.flatten, we no longer get the performance advantage from registered styles."
https://github.com/facebook/react-native/blob/af949877e67c9ff56089436c7980037a9e332c7c/Libraries/StyleSheet/StyleSheet.js#L201
https://github.com/facebook/react-native/blob/af949877e67c9ff56089436c7980037a9e332c7c/Libraries/StyleSheet/flattenStyle.js#L21